src/lsm/groove.zig

```diff
@@ -397,41 +397,47 @@ pub fn GrooveType(
         /// sufficient to query this hashmap alone to know the state of the LSM trees.
         prefetch_objects: PrefetchObjects,
 
+        pub const Options = struct {
+            /// The cache size is meant to be computed based on the left over available memory
+            /// that tigerbeetle was given to allocate from CLI arguments.
+            /// TODO Improve unit in this name to make more clear what should be passed.
+            /// For example, is this a size in bytes or a count in objects? It's a count in objects,
+            /// but the name poorly reflects this.
+            cache_size: u32,
```

Comment:
What about "members" or "entries" or "objects" in place of "cardinality"?

Reply:
`entries` is good. `cardinality` is almost the perfect term though. It's the count of elements in a set.

Reply:
Good point, I'll update it. Just btw, my thinking with `entries` and `objects` was what we could also change `commit_count_max` to `commit_objects_max`, to clarify that it isn't a limit on the number of commits. (Likewise for `prefetch_count_max`).

Reply:
Ah, good point. Shall we do `cache_entries_max`, `commit_entries_max` etc. then?

Rationale being that perhaps `entries` is better than `objects` since secondary indexes don't store objects.